persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -15,7 +15,7 @@ CREATE TABLE Conversation (
     muted_time INTEGER DEFAULT 0 NOT NULL,
     last_modified_date TEXT NOT NULL,
     last_notified_message_date TEXT,
-    last_read_date TEXT,
+    last_read_date TEXT DEFAULT "1970-01-01T00:00:00.000Z" NOT NULL,
```

Comment:
I had assign a default value, because the SQL query was not wokring correctly when the last_read_date was set to NULL getUnreadMessageCount() was returning 0 as count